moveos/moveos-stdlib/moveos-stdlib/sources/storage_context.move

```diff
@@ -33,4 +40,21 @@ module moveos_std::storage_context {
         &mut this.object_storage
     }
 
+    #[test_only]
+    /// Create a StorageContext and AccountStorage for unit test
+    public fun new_test_context(sender: address): StorageContext {
+        let tx_context = tx_context::test_context(sender);
+        // let object_storage = object_storage::new(&mut tx_context);
+        let object_storage = object_storage::new_with_id(@moveos_std_package);
```

Comment:
This address "0x0" is not `moveos_std_package`, I think we should hard code it, and named it `global_object_storage_handle`.